running/ambassador-edge/routing/dark-matter-web/prod.yml

```diff
@@ -0,0 +1,64 @@
+---
+apiVersion: getambassador.io/v2
+kind: Mapping
+metadata:
+  name: darkmatter-prod-mapping
+  namespace: ambassador-edge
+spec:
+  host: darkmatter.prod.apps.mvpstudio.org
+  prefix: /
+  service: darkmatter-web.darkmatter-prod
+  use_websocket: true
+---
+apiVersion: getambassador.io/v2
+kind: Host
+metadata:
+  name: darkmatter-prod-host
+  namespace: ambassador-edge
+spec:
+  hostname: darkmatter.prod.apps.mvpstudio.org
+  acmeProvider:
+    email: darkmatteroregon@gmail.com
+---
+apiVersion: getambassador.io/v2
+kind: Mapping
+metadata:
+  name: darkmatter-bare-dns-mapping
+  namespace: ambassador-edge
+spec:
+  host: darkmatter.consulting
+  prefix: /
+  service: darkmatter-web.darkmatter-prod
+  use_websocket: true
+---
+# Redirect www to non-www
+apiVersion: getambassador.io/v2
+kind: Mapping
+metadata:
+  name: darkmatter-www-dns-mapping
+  namespace: ambassador-edge
+spec:
+  host: www.darkmatter.consulting
+  prefix: /
+  service: darkmatter.consulting
```

Comment:
The service name should always be of the form `<service>.<namespace>`. We don't have a `consulting` namespace and I think you want to create just one service for you server so this should also be `darkmatter-web.darkmatter-prod` once you create the `darkmatter-web` service.

Reply:
updated to dark-matter-web.dark-matter-web-prod  Having service.namespace is confusing to me, but I'll adapt.  I'm used to namespace.bla.